src/test/java/PageObjectStatic/PageStatic/RubberDucksPageStatic.java

```diff
@@ -0,0 +1,89 @@
+package PageObjectStatic.PageStatic;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import java.util.ArrayList;
+
+public class RubberDucksPageStatic {
+
+    WebDriver driver;
+    private static By buttonRubberDarkLocator = By.cssSelector("#site-menu ul > li.category-1");
+    private static By buttonSubCategoryLocator = By.cssSelector("#site-menu ul > li.category-1 > ul > li.category-2 ");
+    private static By wrapperSubCategoryProductsLocator = By.xpath("//li[@class='product row shadow hover-light']");
+    private static By stickerSubCategoryLocator = By.xpath(".//div[contains(@class,'sticker')]");
+    private static By buttonRubberDarkNameLocator = By.xpath("//nav[@class='filter']/a[@class='button'][text()='Name']");
+    private static By productsRubberDarkLocator = By.xpath("//ul[@class='listing-wrapper products']/" +
+            "li[@class='product column shadow hover-light']//div[@class='name']");
+    private static By buttonPriceLocator = By.xpath("//nav[@class='filter']//*[contains(@class,'button')][text()='Price']");
+    private static By productsPriceLocator = By.xpath("//ul[@class='listing-wrapper products']/" +
+            "li[@class='product column shadow hover-light']//span[@class='price']");
+    private static By stickerNewLocator =By.xpath("//div[@class='sticker new']");
+
+
+    public static int duckSubcategory(WebDriver driver) {     // Actions
```

Comment:
please use action-specific method names. currently I don't understand what this method is doing by it's name, I need to look into the method logic which is not good.
Also, no need to start every method name with a "duck" it adds to the confusion